spec/controllers/reports_controller_spec.rb

```diff
@@ -33,19 +33,46 @@
     end
   end
 
+  context 'GET /reports/near/:lat/:long.json' do
+    before do
+      Report.destroy_all
+      3.times do
+        Report.create! valid_create_params
+        Report.create! far_away_create_params
+      end
+    end
+
+    it 'lists only reports near location' do
+      get :index, latitude: 56.878333, longitude: 14.809167
+      expect(json_response.size).to eq(3)
```

Comment:
I'd split the expectations into two blocks instead. A common rule in RSpec is to strive after expecting exactly one thing per `it` block.